EVENT.md

```diff
@@ -199,6 +200,11 @@ received, the browser will find the matching source with highest
 `priority` value and generate a report. The other sources will not
 generate reports.
 
+- `event_report_window`: (optional) duration in seconds relative to the source
```

Comment:
`relative to the source` is a bit ambiguous to me. Maybe `duration in seconds after source registration during which...`?